### PR TITLE
Docker dev env

### PR DESCRIPTION
**Pull request by m0g**
Docker-compose for local dev environment

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
 ================
+
+How to run Liveblog in a full docker environment
+
+```
+cd ~/code/liveblog
+docker-compose -p liveblog -f docker/docker-compose-dev.yml up
+```
+
+SSh into the superdesk docker container:
```
Comment (nistormihai) [inline docker/README.md]: can you provide some easy example on how can ssh into docker?

File: docker/README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ docker-compose -p liveblog -f docker/docker-compose-dev.yml up
 SSh into the superdesk docker container:
 
 ```
+docker exec -i -t `docker ps | grep liveblog_superdesk | awk '{print $1}'` /bin/bash
```
Comment (m0g) [inline docker/README.md]: Not the most straight forward, but this will do it

**Comment by nistormihai** [comment]
LGTM

**Comment by nistormihai** [comment]
please port it also on `devel` branch

**Comment by m0g** [comment]
https://github.com/liveblog/liveblog/pull/482

**Pull request closed without merging**